matchms/filtering/metadata_processing/repair_not_matching_annotation.py

```diff
@@ -84,9 +84,9 @@ def _check_repairing_is_possible(smiles, inchi, inchikey) -> bool:
     elif valid_smiles or valid_inchi or valid_inchikey:
```

Comment:
Now I see this, the valid_inchikey is not enough, since we cannot derive smiles and inchi from inchikey alone. So I would only check elif valid_smiles or valid_inchi: